test/test_boss_resources.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+
+import requests
+import configparser
+
+from boss_resources import *
+# from intern.remote.boss import BossRemote
+
+BOSS_URL = 'https://api.boss.neurodata.io/latest/'
+
+
+class TestBossResources(unittest.TestCase):
+
+    def setUp(self):
+        # create a session for the BOSS using intern
+        self.rmt = BossRemote('neurodata.cfg')
+
+    def tearDown(self):
+        pass
```

Comment:
I would consider adding some code here to delete the resources from the boss (if they exist). We ran into this problem with NDStore tests -- the test would create a test project, fail, and not remove the test project -- and then fail again on subsequent runs because it couldn't create the test project!

Reply:
Just found the setUpClass method which could be useful for this.  Seems better than creating a resource for each test with setUp and deleting it with tearDown.  We will run into problems though because resources are never actually deleted on Boss and cannot have the same name. https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass